src/github.js

```diff
@@ -112,6 +112,17 @@ async function assign_reviewers(reviewers) {
   });
 }
 
+// https://docs.github.com/en/rest/teams/members?apiVersion=2022-11-28#list-team-members
+async function get_team_members(team) {
+  const context = get_context();
+  const octokit = get_octokit();
+
+  return octokit.teams.listMembersInOrg({
+    org: context.repo.org,
+    team_slug: team,
+  })?.map((member) => member.login)
```

Comment:
Have you been able to check if it returns this in this format?
I haven't got around to but I wonder it may return a response in a `data` property 🤔

Reply:
https://docs.github.com/en/rest/teams/members?apiVersion=2022-11-28#list-team-members
It does look like it returns an array. From the documentation the schema doesn't appear to be in a `data` property

Reply:
This is the document for the REST API wrapped in `octokit`, so it doesn't tell us how `octokit` returns its response. It may or may not have `data`. `octokit.repos.getContent` returns a response with a `data` property, so I suspect `octokit.teams.listMembersInOrg` may do the same thing, but I'm not sure yet. Need a way to play around with this API... or the official document (or reading the code...).

https://github.com/necojackarc/auto-request-review/blob/4bb98f3d837b46e4df211fea9857da1e6416e070/src/github.js#L45-L50